Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPCorefinementVisitor.h

```diff
@@ -69,10 +69,10 @@ class PMPCorefinementVisitor{
   /// \param tm_e mesh containing `h_e`
   void intersection_point_detected(std::size_t i_id,
                                    int sdim,
-                                   halfedge_descriptor h_f,
                                    halfedge_descriptor h_e,
-                                   const Triangle_mesh& tm_f,
+                                   halfedge_descriptor h_f,
                                    const Triangle_mesh& tm_e,
+                                   const Triangle_mesh& tm_f,
                                    bool is_target_coplanar,
                                    bool is_source_coplanar);
```

Comment:
- Wouldn't it be better to also change the order of the `\param` commands so they represent the order of the arguments as well.
- doesn't this indicate, again, that it is not a good idea to have the used source and the documentation in separate files.

Reply:
> It's a concept so I don't see how it could be in the code.

I didn't look at the "surrounding" but my comment was based on the fact that the prototype is now at 2 places.

Reply:
Why 2 places? Are you talking of the default model?

Reply:
What I mean, as far as I understand it:
- the part used in the documentation: doc/Polygon_mesh_processing/Concepts/PMPCorefinementVisitor.h at line 72
- the part used in the code: include/CGAL/Polygon_mesh_processing/internal/Corefinement/face_graph_utils.h at line 455

Reply:
That's what I said. These are 2 different things. One is a concept, the other one is the default model.

Reply:
Probably my lack of knowledge about concepts.